tests/functional/adapter/list_relations_tests/test_pagination.py

```diff
@@ -1,19 +1,22 @@
+import os
+
 import pytest
 
 import json
 from dbt.tests.util import run_dbt, run_dbt_and_capture
 
 # Testing rationale:
 # - snowflake SHOW TERSE OBJECTS command returns at max 10K objects in a single call
-# - when dbt attempts to write into a scehma with more than 10K objects, compilation will fail
+# - when dbt attempts to write into a schema with more than 10K objects, compilation will fail
 #   unless we paginate the result
 # - however, testing this process is difficult at a full scale of 10K actual objects populated
 #   into a fresh testing schema
 # - accordingly, we create a smaller set of views and test the looping iteration logic in
 #   smaller chunks
 
-NUM_VIEWS = 100
-NUM_EXPECTED_RELATIONS = 1 + NUM_VIEWS
+NUM_VIEWS = 90
```

Comment:
Convert 10 views to dynamic tables to keep the same number of total relations. Changing the number affects the pagination scenarios.

Reply:
Can we comment the magic number? I still don't know the purpose of pagination at 100. Like what behavior are we teasing out exactly? Would be nice to know / be readable for future folks.

Reply:
I believe the magic number is 110, based on this test [macro](https://github.com/dbt-labs/dbt-snowflake/blob/87a6e808dfb025df1eeef3741ad3822635249889/tests/functional/adapter/test_list_relations_without_caching.py#L30). I originally just added 10 dynamic tables and wound up with the "passing" macro failing (100 views, 10 dynamic tables, 1 seed). I figured it was due to the number of objects so I decided to keep that a constant. I could add a comment in here pointing to that line in the macro definition below if so desired.

Reply:
I added the comment below.